client-sdk/ts-web/webpack.config.js

```diff
@@ -0,0 +1,19 @@
+const webpack = require('webpack');
+
+module.exports = {
+    mode: 'development',
+    output: {
+        library: {
+            name: 'playground',
+            type: 'window',
+            export: 'playground',
+        },
+    },
+    resolve: {fallback: {stream: require.resolve('stream-browserify')}},
+    plugins: [
+        new webpack.ProvidePlugin({
+            process: 'process/browser.js',
+            Buffer: ['buffer', 'Buffer'],
```

Comment:
I hadn't seen it done this way. maybe we could use this to let ledger-js compile with webpack 5. all the docs on webpack say is "ya'd better make sure your stuff works without process and Buffer before you choose to use webpack 5" :sweat: 

Could we have these polyfills isolated to where it's needed, rather than in this shared config file? Stream in `core` and process and Buffer in `signer-ledger`. Possibly Buffer in `core`, although I've left a comment on the possibility of that not being needed there.

Reply:
Stream is actually required in all of them - but since the configs have changed a bit between them and I'm not aware of the specifics I added them separately

Reply:
Actually, we still need buffer because of bip39, and in all three packages

```
  > Buffer is not defined

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application
      at eval (webpack://playground/../../node_modules/core-util-is/lib/util.js?:103:20)
      at Object.../../node_modules/core-util-is/lib/util.js (http://localhost:42280/main.js:676:1)
      at __webpack_require__ (http://localhost:42280/main.js:1648:32)
      at fn (http://localhost:42280/main.js:1848:21)
      at eval (webpack://playground/../../node_modules/readable-stream/lib/_stream_readable.js?:67:26)
      at Object.../../node_modules/readable-stream/lib/_stream_readable.js (http://localhost:42280/main.js:1144:1)
      at __webpack_require__ (http://localhost:42280/main.js:1648:32)
      at fn (http://localhost:42280/main.js:1848:21)
      at eval (webpack://playground/../../node_modules/readable-stream/readable-browser.js?:1:28)
      at Object.../../node_modules/readable-stream/readable-browser.js (http://localhost:42280/main.js:1228:1)
```